rest/replicator_test.go

```diff
@@ -3933,6 +3932,23 @@ func TestSGR2TombstoneConflictHandling(t *testing.T) {
 	}
 
 	for _, test := range tombstoneTests {
+		// requireTombstone validates tombstoned revision.
+		requireTombstone := func(bucket *base.TestBucket, docID string) {
```

Comment:
Does this function need to be redefined for each test? I can't see anything inside it using `test` for it to be warranted.

More generally, is it useful enough to be pulled outside of this test for re-use?

Reply:
That's a fair point, this function doesn't need to be redefined inside each test. The main main motivation for this writing this function is reusability and it should be outside each test.